### PR TITLE
CI: add globbing to ignore

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Run tests
-        run: docker run -v $PWD:/workdir ghcr.io/igorshubovych/markdownlint-cli:latest --ignore ./storage/v1alpha1/autogen.md --ignore ./security/v1/autogen.md --ignore ./network/v1/autogen.md --ignore ./common/v1/autogen.md --disable=MD013 "**/*.md"
+        run: docker run -v $PWD:/workdir ghcr.io/igorshubovych/markdownlint-cli:latest --ignore "./*/*/autogen.md" --ignore "./network/*/*/autogen.md" --disable=MD013 "**/*.md"
```
Comment (seroyer) [inline .github/workflows/linters.yml:16]: does `--ignore "**/autogen.md"` not work?

Comment (glimchb) [reply .github/workflows/linters.yml:16]: let me try...

Comment (glimchb) [reply .github/workflows/linters.yml]: works!

**Review by seroyer — APPROVED** [review]
Better, thanks!

**Pull request merged**